frontend/src/components/SidebarComponent.vue.js

```
@@ -299,7 +299,34 @@ function __VLS_template() {
+        ...{ class: (({ active: __VLS_ctx.$route.path === '/ueber-uns' })) },
+    }, ...__VLS_functionalComponentArgsRest(__VLS_55));
+    __VLS_elementAsFunction(__VLS_intrinsicElements.div, __VLS_intrinsicElements.div)({
+        ...{ class: ("icon icon-shape icon-sm shadow border-radius-md text-center me-2 d-flex align-items-center justify-content-center") },
+    });
+    __VLS_elementAsFunction(__VLS_intrinsicElements.i, __VLS_intrinsicElements.i)({
+        ...{ class: ("material-icons opacity-10") },
+    });
+    __VLS_elementAsFunction(__VLS_intrinsicElements.span, __VLS_intrinsicElements.span)({
+        ...{ class: ("nav-link-text ms-1") },
+    });
+    __VLS_59.slots.default;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)